tests/bsim/bluetooth/tester/src/bsim_btp.c

```diff
@@ -31,6 +31,7 @@ K_FIFO_DEFINE(btp_evt_fifo);
 NET_BUF_POOL_FIXED_DEFINE(btp_evt_pool, 100, BTP_MTU, 0, NULL);
 
 static const struct device *const dev = DEVICE_DT_GET(DT_CHOSEN(zephyr_console));
+static volatile uint8_t last_send_opcode;
```

Comment:
Is the `volatile` needed? I think `volatile` does not do anything useful here (based on what the C standard specifies).